docs/examples/generate_diffs.sh

```diff
@@ -39,4 +43,8 @@ generate_diff distributed/single_gpu/main.py good_practices/checkpointing/main.p
 generate_diff distributed/single_gpu/job.sh good_practices/hpo_with_orion/job.sh
 generate_diff distributed/single_gpu/main.py good_practices/hpo_with_orion/main.py
 
+# single_gpu -> jax
+generate_diff distributed/single_gpu/job.sh frameworks/jax/job.sh
+generate_diff distributed/single_gpu/main.py frameworks/jax/main.py
+
```

Comment:
I'm still not convinced that users would find it useful to see the Jax examples as diffs w.r.t. to PyTorch examples.
This to me sounds the same as doing a TensorFlow 1.0 -> PyTorch diff. Might as well start from scratch imo.

Reply:
Semi done ;). I've kept the job.sh.diff since why not I suppose?